arbnode/node.go

```diff
@@ -615,18 +615,22 @@ func createNodeImpl(
 		}
 	}
 
+	l2ChainId := l2BlockChain.Config().ChainID.Uint64()
 	var classicOutbox *ClassicOutboxRetriever
 	classicMsgDb, err := stack.OpenDatabase("classic-msg", 0, 0, "", true)
 	if err != nil {
-		log.Warn("Classic Msg Database not found", "err", err)
+		if l2ChainId == 42161 || l2ChainId == 421611 {
+			// we only expect classic msg on rinkeby and arbitrum one
+			log.Warn("Classic Msg Database not found", "err", err)
+		}
```

Comment:
I'm wondering if we can use whether the genesis block is greater than 0 for this?